Projects/Tweeter/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
each project should be its own repository on github. Do not put all your projects in one project folder